### PR TITLE
Remove some trivial differences in VS201x backend

### PR DESCRIPTION
**Pull request by vadz**
This is mainly to facilitate comparing bakefile-generated projects with the original, IDE-created, ones.

### CODE REVIEW

File: src/bkl/plugins/vs201x.py [hunk 1]
```diff
@@ -348,7 +359,7 @@ def gen_for_target(self, target, project):
         filename = project.projectfile.as_native_path_for_output(target)
         paths_info = self.get_project_paths_info(target, project)
 
-        formatter = XmlFormatter(target.project.settings, paths_info)
+        formatter = VS201xXmlFormatter(target.project.settings, paths_info)
```
Comment (vslavik) [inline src/bkl/plugins/vs201x.py]: Why? `XmlFormatter` should already be this one (it’s a class variable), using it seems safer - if some future derived class needs further tweaks, they’ll be picked up automatically.

Comment (vadz) [inline src/bkl/plugins/vs201x.py]: `XmlFormatter` wasn't set correctly, the latest commit should fix this, thanks.

**Comment by vslavik** [comment]
I see nothing wrong with the code (I assume you verified the output), thanks!

The only potential issue with the `format_node` refactoring I can see is performance due to so many function calls. But I really have no idea if that can have meaningful impact in modern (2.7) Python, just saying that it’s IMHO work testing on a big project vs the previous version.

**Comment by vadz** [comment]
FWIW I don't see any difference with a big project under Windows, albeit using PyPy.

**Pull request closed without merging**